tests/5.1/target/test_target_memcpy_rect_async_no_obj.c

```diff
@@ -68,8 +71,9 @@ int test_target_memcpy_async_depobj() {
                                 h,          t,
                                 0,          NULL);
 
+    #pragma omp taskwait
```

Comment:
Why do we need this taskwait ?

Reply:
Per 5.1 spec pg. 425 starting from line 25: "The omp_target_memcpy_rect_async routine executes as if part of a target task that is generated by the call to the routine and for which execution may be deferred" which from my guess means that the arrays hostRect and devRect could perhaps be accessed before the   omp_target_memcpy_rect_async() call specifically allocates memory for them. So, the taskwait ensures that any tasks before the pragma are executed.